ipa-core/src/report.rs

```diff
@@ -56,8 +56,8 @@ impl Serializable for EventType {
 
     fn serialize(&self, buf: &mut GenericArray<u8, Self::Size>) {
         let raw: &[u8] = match self {
-            EventType::Trigger => &[0],
-            EventType::Source => &[1],
+            EventType::Trigger => &[1],
+            EventType::Source => &[0],
```

Comment:
I think it will be an explicit merge conflict, but just a heads up that this may interact with the fallible serialization PR.